suite-native/module-accounts-management/src/screens/AccountsScreen.tsx

```diff
@@ -49,8 +49,9 @@ export const AccountsScreen = () => {
                     title="My assets"
                     onSearchInputChange={handleFilterChange}
                     rightIcon={
-                        isPortfolioTrackerDevice &&
-                        !areAllDevicesDisconnectedOrAccountless && <AddAccountButton />
+                        (isPortfolioTrackerDevice || !areAllDevicesDisconnectedOrAccountless) && (
```

Comment:
Could be simplified to `!areAllDevicesDisconnectedOrAccountless && <Cmp />`, no?